main.js

```diff
@@ -42,6 +40,45 @@ let retrieveStats = () => {
     });
 };
 
+let retrieveCompanies = () => {
+    let companyList = document.querySelector('[name="companies-dropdown-menu"]'); 
+    getPromise = fetch(`${urlAPI}companyList/?token=${token}`,
+        {
+            method: "get"
```

Comment:
Get is the default, you don't actually need this here. It's not hurting anything though.